model/src/w3gdatmd.F90

```diff
@@ -700,6 +701,7 @@ MODULE W3GDATMD
 
     LOGICAL          :: GINIT, FLDRY, FLCX, FLCY, FLCTH, FLCK, FLSOU, IICEDISP,&
          IICESMOOTH
+    LOGICAL, POINTER :: IC_NUMERICS
```

Comment:
Is this supposed to be a pointer in both places?

Reply:
I'm not 100% sure to be honest - I don't understand why some of the namlist settings here are pointers... but other are not... for example, most of the IC4 namelist settings are pointers in both places? but the IICEDISP is only a pointer in one place?

Reply:
@erinethomas @dabail10 While trying to test this PR, my runs failed in ww3_grid. After debugging I figured that the IC_NUMERICS should be just a logical in line 704 and pointer reference on line 1221. I changed the code accordingly and it git passed that error. I have a hard time understanding why the pointers should be defined like this but I can confirm that without this change the code does not run on at least the machine I was trying to run it on (Hera).
@erinethomas you did mention the IC4 namelists, but I guess since they are only complied within an #ifdef, they don't have to be defined the same way as MISC namelists? (or at least that's the only difference I see between IC_NUMERICS or IICEDISP and IC4 namelists.)

Reply:
@sbanihash - ah I see. perhaps that is why. I will update the PR to reflect these changes (logical on line 704, pointer on line 1221)

Reply:
Thank you @erinethomas. Do you get the same error when running? (I mean without this change?) because I see in the discussions that @dabail10 says changes work in CESM code.